jekyll/_cci2/rotate-project-ssh-keys.adoc

```diff
@@ -0,0 +1,110 @@
+
+
+[#rotate-a-user-key-bitbucket]
+=== Rotate a user key
+
+. Take note of the current key information to rotate.
+. Delete the deploy key by clicking the **X**.
+. Add a new user key following the xref:bitbucket-integration/#create-a-bitbucket-user-key[Create a Bitbucket user key] instructions.
+. Go to Bitbucket’s user account settings to delete the matching public key. The Bitbucket URL is typically `https://bitbucket.org/account/settings/ssh-keys/`. The user names the keys, therefore, CircleCI does not know if the key name contains the string `CircleCI`. It is recommended to remove any key created before the rotation.
+
+[#rotate-an-additional-SSH-key-bitbucket]
+=== Rotate an additional SSH key
+
+. Take note of the existing key to know which target system it is used for. This could be a version control system, a machine or other SSH based system.
+. Delete the deploy key by clicking the **X**.
+. Remove the authorization to use that key (the public key) from the target system. Typically, the public key will be authorized in the target system, and the configuration varies depending on the target systems.
+. Recreate the key and the authorization in the target system.
+
+[#gitlab-projects]
+== GitLab projects
+
+Go to **Project Settings > SSH Keys** to view SSH keys set up for your project. The GitLab integration with CircleCI does not use deploy keys or user keys.
+
+[#rotate-an-SSH-key-gitlab]
+=== Rotate an SSH key
+
+. Take note of the existing key and fingerprint to be able to match the target system it is used for, typically `gitlab.com`.
+. Delete the deploy key by clicking the **X**.
+. The key will need to be recreated manually, for example, using OpenSSH (`ssh-keygen -t ed25519 -C "your_email@example.com"`). Instructions can be found on link:https://docs.gitlab.com/ee/user/ssh.html[GitLab's documentation].
+. Add the private key by clicking **Add SSH Key**. It is recommended you name the key `gitlab.com`. Paste in the private key you created in the step above and click **Add SSH Key** again to finalize.
+. Go to GitLab’s project settings for the repository and expand deploy keys section. The GitLab URL is typically `https://gitlab.com/<org>/<project>/-/settings/repository`. The keys are named `circleci-pipeline-triggers` and you can match the fingerprint. It is recommended to remove any key created before the rotation.
+. Add the new key. For consistency, you can name the key in the title section `circleci-pipeline-triggers`. Paste the public key and click **Add key** to finalize.
+
+[#rotate-an-additional-SSH-key-gitlab]
+=== Rotate an additional SSH key
+
+In some scenarios, you might also be using additional SSH keys that are not tied to GitLab.
+
+. Take note of the existing key to know which target system it is used for. This could be a version control system, a machine or other SSH based system.
```

Comment:
```suggestion
. Take note of the existing key's Hostname to know which target system it is used for and the fingerprint for your records. This could be a version control system, a machine or other SSH based system.
```